gen/config/alertsource.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
what are these files?

Reply:
@rocktavious these files are the output of `task parse` reading our opslevel-go functions and extracting all the details from them. A generator for a program like CLI can consume these .json files and use templates to generate code that uses opslevel-go.

Reply:
long term the data should beable to be built up in memory from calling a function exposed by the library so if something in the CLI needs them it can just call the code machinery exposed by opslevel-go to do this.

Having a bunch of "data files" in a library is generally architecture smell to me unless they are test fixtures.

Reply:
@rocktavious The problem with the data not being serialized somewhere is that we would not be able to see changes in between versions.  That makes it much easier to track what changes in opslevel-go are affecting other repos and makes it easier to see what someone ended up changing in a pull request.  

I'm working on narrowing down the number of files generated since it is annoying. Working on a change now to make there only be 2 files: `generated.json` and `custom.json`, where `generated.json` is what the parser outputted and `custom.json` is full of any overrides we have set.

Reply:
❤️  - i can get behind the centralized file as a middle ground.

Reply:
Changed code to do this in memory - no more files needed. Lets us cut down on the # of lines of code too, and writing file IO logic is annoying.